### PR TITLE
test

### PR DESCRIPTION
**Pull request by Silvia02**
#test

### CODE REVIEW

**Comment by Silvia02** [comment]
fixet

File: collaboration.html [hunk 1]
```diff
@@ -5,15 +5,31 @@
...
 
 <body>
     <div>
         <h1>hej!</h1>
         <h2>hallå!</h2>
         <h3>Hallåhej!</h3>
+
+        <h4>feature 1, feature 2</h4>
+
         <h4>Hello hey! what's your name?</h4>
```
Comment (oscarlundberg-sthlm) [inline collaboration.html:27]: ska vi ta bort "feature1, feature 2"?

Comment (Silvia02) [reply collaboration.html:27]: yes!

**Comment by Silvia02** [comment]
#Nav-bar done

**Pull request merged**